### PR TITLE
Add info about C2 cluster

### PR DESCRIPTION
**Pull request by nilsgstrabo**
## What type of PR? (can choose many)
- [x] 🍕 Added/Updated documentation
- [ ] 🎨 Style
- [ ] 🔒 Security

## Description

- Reduce memory and cpu request. The static files are now served with NGINX instead of with docusaurus. NGINX requires less resources.

### CODE REVIEW

File: public-site/docs/docs/topic-domain-names/index.md [hunk 1]
```diff
@@ -4,95 +4,100 @@ title: Domain names
...
-
-Examples:
-
-- `frontend-myapp-production.playground-92.playground.radix.equinor.com`
-- `backend-myapp-production.playground-92.playground.radix.equinor.com`
-- `serializer-oneapp-qa.eu-12.radix.equinor.com`
+- `component-name`: The name of the component, e.g. `frontend`.
+- `app-name`: The name of the application, e.g. `myapp`.
+- `env-name`: The name of the environment where the component is deployed to, e.g. `production`.
+- `cluster-name`: The name of the underlying Kubernetes cluster used for hosting a specific [Radix cluster](../../start/radix-clusters/), e.g. `eu-18`. This value can change, for example during upgrade of a Radix cluster. Domain names using this value should only be used for debugging purposes, and should never be used by end users/services.
```
Comment (satr) [inline public-site/docs/docs/topic-domain-names/index.md:14]:
```suggestion
- `cluster-name`: Exists only in [canonical name](./#canonical-name).  The name of the underlying Kubernetes cluster used for hosting a specific [Radix cluster](../../start/radix-clusters/), e.g. `eu-18`. This value can change, for example during upgrade of a Radix cluster. Domain names using this value should only be used for debugging purposes, and should never be used by end users/services.
```

File: public-site/docs/docs/topic-domain-names/index.md [hunk 2]
```diff
@@ -4,95 +4,100 @@ title: Domain names
...
+- `cluster-dns-zone`: The DNS zone for the [Radix cluster](../../start/radix-clusters/) where the application is hosted, e.g. `radix.equinor.com`.
```
Comment (satr) [inline public-site/docs/docs/topic-domain-names/index.md:15]:
```suggestion
- `cluster-dns-zone`: The DNS zone for the [Radix cluster](../../start/radix-clusters/) where the application is hosted, e.g. `radix.equinor.com`, `playground.radix.equinor.com`.
```

File: public-site/docs/docs/topic-domain-names/index.md [hunk 3]
```diff
@@ -4,95 +4,100 @@ title: Domain names
...
 
 ## Public name
 
 ```text
-[componentName]-[appName]-[envName].[clusterEnvNamepace].radix.equinor.com
+[component-name]-[app-name]-[env-name].[cluster-dns-zone]
 ```
 
 - Automatically allocated
 - One per component
```
Comment (satr) [inline public-site/docs/docs/topic-domain-names/index.md:24]:
```suggestion
- Unique for each component in each environment
```
or
```suggestion
- One per a component, per an environment
```
similar for other "Multiple allowed per component" and "One per component"

File: public-site/docs/docs/topic-domain-names/index.md [hunk 4]
```diff
@@ -4,95 +4,100 @@ title: Domain names
...
+[a valid external DNS name]
 ```
 
 For ultimate customisation of your domain names, you can "bring your own" domain into Radix with an _external alias_. There is a [detailed guide](/guides/external-alias/) on how to configure this.
 
 - Multiple allowed per component
 - [Defined in `radixconfig.yaml`](/radix-config/index.md#dnsexternalalias)
 - Requires external DNS alias management
-- Requires custom TLS certificate
+- Bring your own TLS certificate, or let Radix handle it
```
Comment (satr) [inline public-site/docs/docs/topic-domain-names/index.md:78]:
```suggestion
- Bring your own TLS certificate, or let Radix to handle it
```
Should we add a link to docs where it is described how Radix can handle it?

**Review by satr — APPROVED** [review]

File: public-site/docs/docs/topic-domain-names/index.md (hunk 4 above)
Comment (nilsgstrabo) [reply public-site/docs/docs/topic-domain-names/index.md]: added links to manual and automatic TLS certs

**Review by satr — APPROVED** [review]

**Pull request merged**